doc/sphinxext/custom_roles.py

```diff
@@ -3,7 +3,15 @@
 
 def rcparam_role(name, rawtext, text, lineno, inliner, options={}, content=[]):
     rendered = nodes.Text('rcParams["{}"]'.format(text))
-    return [nodes.literal(rawtext, rendered)], []
+
+    rel_source = inliner.document.attributes['source'].split('/doc/', 1)[1]
+    levels = rel_source.count('/')
+    refuri = ('../' * levels +
+              'tutorials/introductory/customizing.html#matplotlib-rcparams')
+
+    ref = nodes.reference(rawtext, rendered, refuri=refuri,
+                          classes=['rcparams'])
```

Comment:
would perhaps not bother with the separate class

Reply:
Doesn't harm. And makes it easier to adapt the style of the link if desired (now or later).

Reply:
(that comment basically just applies if we decide not to style rcs differently from normal links)